index.d.ts

```diff
@@ -4,7 +4,7 @@
 /**
  * Container for all registered metrics
  */
-export interface register {
+export interface Registry {
```

Comment:
Ok, thanks. Probably later I'll have some time to check it with some projects in written typescript to be sure that all is fine.